src/test/kotlin/com/hibob/academy/employee_feedback_feature/dao/FeedbackDaoTest.kt

```diff
@@ -0,0 +1,122 @@
+package com.hibob.academy.employee_feedback_feature.dao
+
+import com.hibob.academy.utils.BobDbTest
+import org.jooq.DSLContext
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import org.springframework.beans.factory.annotation.Autowired
+import org.junit.jupiter.api.Test
+
+@BobDbTest
+class FeedbackDaoTest @Autowired constructor(private val sql: DSLContext) {
+    private val companyId = 3L
+    private val employeeId = 1L
+    private val feedbackText = "What a feedback!"
+    private val isAnonymous = false
+    private val department = "Sales"
+    private val testFeedback = FeedbackSubmission(employeeId, companyId, feedbackText, isAnonymous, department)
+    private val dao = FeedbackDao(sql)
+    val feedback = FeedbackTable.instance
+
+    @BeforeEach
+    @AfterEach
+    fun cleanup() {
+        sql.deleteFrom(feedback).where(feedback.companyId.eq(companyId)).execute()
+    }
+
+
+    @Test
+    fun `Submit new feedback- not anonymous`() {
+        val submittedFeedbackId = dao.submitFeedback(testFeedback)
+        assertNotNull(submittedFeedbackId)
+    }
+
+    @Test
+    fun `Submit new feedback - anonymous submission`() {
+        val anonymousFeedback = testFeedback.copy(isAnonymous = true)
+        val submittedFeedbackId = dao.submitFeedback(anonymousFeedback)
+        assertNotNull(submittedFeedbackId)
+    }
+
+    @Test
+    fun `Get feedback history - feedback exists`() {
+        dao.submitFeedback(testFeedback)
+
+        val feedbackHistory = dao.getFeedbackHistory(companyId, employeeId)
```

Comment:
what about other employees?

Reply:
I have getAllFeedbacks for that scenario

Reply:
I mean that maybe add data for another employee and then the test shows that you didn't fetch it